### ISSUE TITLE
database creation fails

### ISSUE THREAD

**Issue by RobTranquillo** [shopwareLabs/shopware-docker#21]
I'am on fedora 28, Docker version 1.13.1 and try to get this repo working. But  I'm stuck while creating database fail inside the ssh session.

` app-shell@acbce5b77356:/var/www/shopware$ ./psh.phar init `
<pre>
         [...]
         37376/37393 [===========================>]  99%
         37393/37393 [============================] 100%
         Download finished
        Unzipping archive
        Creating database shopware

                                                                                                                                                                        
          [PDOException]                                                                                                                                                
          SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known                                                                
                                                                                                                                                                        

                                                                                                                                                                        
          [PDOException]                                                                                                                                                
          PDO::__construct(): php_network_getaddresses: getaddrinfo failed: Name or service not known                                                                   
                                                                                                                                                                        

        install:release [-r|--release RELEASE] [-i|--install-dir INSTALL-DIR] [--db-host DB-HOST] [--db-port DB-PORT] [--db-socket DB-SOCKET] [--db-user DB-USER] [--db-password DB-PASSWORD] [--db-name DB-NAME] [--no-skip-import] [--shop-locale SHOP-LOCALE] [--shop-host SHOP-HOST] [-p|--shop-path SHOP-PATH] [--shop-name SHOP-NAME] [--shop-email SHOP-EMAIL] [--shop-currency SHOP-CURRENCY] [--skip-admin-creation] [--admin-username ADMIN-USERNAME] [--admin-password ADMIN-PASSWORD] [--admin-email ADMIN-EMAIL] [--admin-locale ADMIN-LOCALE] [--admin-name ADMIN-NAME]


Execution aborted, a subcommand failed!
</pre>

**Comment by bobbyshaw** [comment]
For me, I found that the app_mysql container was no longer running by the time the init command was trying to create the database. In fact, it had exited immediately on boot.

I was able to debug my own problem by running `docker-compose logs` optionally with the `-f` option for following the logs as I tried to restart. This showed me that the database hadn't recovered after my VM ran out of disk space. I was able to resolve by removing the MySQL container with:

```
docker-compose rm app_mysql
```

I then restarted the process with
```
./psh.phar docker:start
```

I was then able to successfully complete the following commands:

```
./psh.phar docker:ssh
./psh.phar init-vcs --sw-branch=5.5
```

@RobTranquillo Your problem is unlikely to be space but hopefully some of those steps will help you to debug the problem if it's still of value for you to do so.
